src/index.js

```diff
@@ -1,10 +1,25 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { render } from 'react-dom';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
```

Comment:
same as with the other pair, it would be nice to organize the imports: https://dev.to/otamnitram/sorting-your-imports-correctly-in-react-213m just a guide, no need to add eslint

Reply:
Thanks, I'm a little confused about sort services worker, [5cbe2737646b5ab919c587cd58e06220b6fe0419](https://github.com/the-collab-lab/tcl-39-smart-shopping-list/commit/5cbe2737646b5ab919c587cd58e06220b6fe0419)